hack/test-update-storage-objects.sh

```diff
@@ -63,6 +63,9 @@ function startApiServer() {
 function killApiServer() {
   kube::log::status "Killing api server"
   [[ -n ${APISERVER_PID-} ]] && kill ${APISERVER_PID} 1>&2 2>/dev/null
+
+  wait ${APISERVER_PID} || true
```

Comment:
I think you need a `[[ -n ${APISERVER_PID-} ]] ||` here, too, in case this is called when ${APISERVER_PID} is not set?